run/GCClassic/input_options.yml.templates/input_options.yml.fullchem

```diff
@@ -0,0 +1,426 @@
+      - HCFC22
+      - HCl
+      - HCOOH
+      - HI
+      - HMHP
+      - HMML
+      - HMS
+      - HNO2
+      - HNO3
+      - HNO4
+      - HOBr
+      - HOCl
+      - HOI
+      - HONIT
+      - HPALD1
+      - HPALD2
+      - HPALD3
+      - HPALD4
+      - HPETHNL
+      - I
+      - I2
+      - I2O2
+      - I2O3
+      - I2O4
+      - IBr
+      - ICHE
+      - ICl
+      - ICN
+      - ICPDH
+      - IDC
+      - IDCHP
+      - IDHDP
+      - IDHPE
+      - IDN
+      - IEPOXA
+      - IEPOXB
+      - IEPOXD
+      - IHN1
+      - IHN2
+      - IHN3
+      - IHN4
+      - INDIOL
+      - INO
+      - INPB
+      - INPD
+      - IO
+      - IONITA
+      - IONO
+      - IONO2
+      - IPRNO3
+      - ISALA
+      - ISALC
+      - ISOP
+      - ITCN
+      - ITHN
+      - LIMO
+      - LVOC
+      - LVOCOA
+      - MACR
+      - MACR1OOH
+      - MAP
+      - MCRDH
+      - MCRENOL
+      - MCRHN
+      - MCRHNB
+      - MCRHP
+      - MCT
+      - MEK
+      - MENO3
+      - MGLY
+      - MOH
+      - MONITA
+      - MONITS
+      - MONITU
+      - MP
+      - MPAN
+      - MPN
+      - MSA
+      - MTPA
+      - MTPO
+      - MVK
+      - MVKDH
+      - MVKHC
+      - MVKHCB
+      - MVKHP
+      - MVKN
+      - MVKPC
+      - N2O
+      - N2O5
+      - NH3
+      - NH4
+      - NIT
+      - NITs
+      - 'NO'
+      - NO2
+      - NO3
+      - NPHEN
+      - NPRNO3
+      - O3
+      - OClO
+      - OCPI
+      - OCPO
+      - OCS
+      - OIO
+      - PAN
+      - pFe
+      - PHEN
+      - PIP
+      - PP
+      - PPN
+      - PROPNN
+      - PRPE
+      - PRPN
+      - PYAC
+      - R4N2
+      - R4P
+      - RA3P
+      - RB3P
+      - RCHO
+      - RIPA
+      - RIPB
+      - RIPC
+      - RIPD
+      - RP
+      - SALA
+      - SALAAL
+      - SALACL
+      - SALC
+      - SALCAL
+      - SALCCL
+      - SO2
+      - SO4
+      - SO4s
+      - SOAGX
+      - SOAIE
+      - SOAP
+      - SOAS
+      - TOLU
+      - XYLE
+
+  wet_deposition:
+    activate: true
+
+#============================================================================
+# Settings for GEOS-Chem aerosols
+#============================================================================
+aerosols:
+
+  carbon:
+    activate: true
+    brown_carbon: false
+    enhance_black_carbon_absorption:
+      activate: true
+      hydrophilic: 1.5
+      hydrophobic: 1.0
+
+  complex_SOA:
+    activate:  ${RUNDIR_COMPLEX_SOA}
+    semivolatile_POA: ${RUNDIR_SVPOA}
+
+  dust:
+    activate: true
+    acid_uptake_on_dust: ${RUNDIR_ACID_UPTAKE}
+
+  sea_salt:
+    activate: true
+    SALA_radius_bin_in_um: [0.01, 0.5]
+    SALC_radius_bin_in_um: [0.5,  8.0]
+    marine_organic_aerosols: ${RUNDIR_MARINE_POA}
+
+  stratosphere:
+    settle_strat_aerosol: true
+    polar_strat_clouds:
+      activate: true
+      het_chem: true
+    allow_homogeneous_NAT: false
+    NAT_supercooling_req_in_K: 3.0
+    supersat_factor_req_for_ice_nucl: 1.2
+    calc_strat_aero_optdepth: true
```

Comment:
maybe calc_strat_aod for the sake of consistency with `aod_wavelengths_in_nm`?

Reply:
Now fixed.

Reply:
This is now fixed in https://github.com/geoschem/geos-chem/commit/a6cc6617a6ecb9597e8d39ce7ceff0eb880739e4